File: _tmp_check.js
const fs = require('fs');
const html = fs.readFileSync(
  'c:/Users/28312/WorkBuddy/2026-05-18-task-15/deliverables/bank-stock-system.html',
  'utf-8'
);
const scripts = html.match(/<script>([\s\S]*?)<\/script>/g) || [];
scripts.forEach((s, i) => {
  const code = s.replace(/<\/?script>/g, '');
  if (code.length > 100) {
    try {
      new Function(code);
      console.log(`Script ${i}: ${code.split('\n').length} lines - SYNTAX OK`);
    } catch(e) {
      console.log(`SCRIPT ${i} SYNTAX ERROR: ${e.message}`);
    }
  }
});

File: _tmp_check.py
import re
f=open(r'c:\Users\28312\WorkBuddy\2026-05-18-task-15\deliverables\bank-stock-system.html', encoding='utf-8')
c=f.read()
scripts=re.findall(r'<script>(.*?)</script>', c, re.DOTALL)
for i,s in enumerate(scripts):
    if len(s) > 100:
        try:
            compile(s, f'script_{i}.js', 'exec')
            lines = s.count('\n')
            print(f'Main script ({lines} lines): SYNTAX OK')
        except SyntaxError as e:
            print(f'MAIN SCRIPT SYNTAX ERROR at line {e.lineno}: {e.msg}')
            print(f'  Context: {e.text[:100] if e.text else "N/A"}')
